### PR TITLE
Update XBlock and xblock-sdk depenencies

### PR DESCRIPTION
**Pull request by muhammad-ammar**
TNL-1068

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@symbolist Can you please review?

**Comment by muhammad-ammar** [comment]
@symbolist Done

File: requirements/test-acceptance.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-bok_choy==0.3.1
-nose==1.3.0
+-e git+https://github.com/edx/bok-choy.git@4a259e3548a19e41cc39433caf68ea58d10a27ba#egg=bok_choy
```
Comment (symbolist) [inline requirements/test-acceptance.txt]: I think this does not need to be this exact. The latest version 0.3.2 would do fine.

**Comment by symbolist** [comment]
Besides the comment about bok_choy :+1:

**Pull request merged**